lib/solidus_dev_support/rake_tasks.rb

```diff
@@ -44,6 +44,7 @@ def install_test_app_task
         end
```

Comment:
@waiting-for-dev In SolidusPaypalCommercePlatform's case, `the extension:test_app` wrapper task defined in `lib/solidus_dev_support/rake_tasks.rb` is invoked _after_ `Solidus::InstallGenerator` runs. To recap, here are all the tasks and Ruby methods executed by `bundle exec rake` in SolidusPaypalCommercePlatform, including the `extension:test_app` wrapper task.

1. The `extension` namespace defined in `SolidusDevSupport::RakeTasks#install_test_app_task`, which invokes
  the `extension:test_app` task.
2. The Solidus `extension:test_app` task
3. The `common:test_app` rake task in `CommonRakeTasks#initialize`
4. `Solidus::InstallGenerator`
5. The `extension:test_app` wrapper task in SolidusDevSupport (the `test_app` task above)
6. The Solidus `extension:test_app` task again
7. `extension:specs`, which is the default task of SolidusPaypalCommercePlatform

I'm guessing that the `extension:test_app` wrapper task (Step 5 above) is called as part of the whole invocation of the `extension:test_app` task. That invocation calls the "original" Solidus `extension:test_app` task first, before calling the overriding `extension:test_app` task from SolidusDevSupport.

Considering that Step 1 has already set up `ENV['RAILS_ENV'] = 'test'`, there is no need to do the same for the `extension:test_app` wrapper task (Step 5 above), _except_ to make it consistent with the change I made to `Rake::Task['extension:test_app'].invoke` below. Let me know if you prefer to err on the side of consistency.

Finaly, note  that when the Solidus `extension:test_app` task is invoked a second time (in Step 6), it doesn't do anything. I still have to check but I'm guessing this is a normal behavior for Rake: it runs a task only once.

I'm making several guesses here concerning Rake, so I'll need to review Rake's documentation, especially with regard to how it handles overridden rake tasks.